packages/atomic/src/components/commerce/facets/facet-number-input/atomic-commerce-facet-number-input.tsx

```diff
@@ -59,8 +59,20 @@ export class FacetNumberInput {
     ]);
   }
 
+  private get minimumInputValue() {
+    const {domain} = this.facet.state;
+    const min = Number.MIN_SAFE_INTEGER;
+    return Math.max(this.start || min, domain?.min || min);
```

Comment:
I think the falsy check will stumble if `this.start` or `domain.min` is 0.
We should do strict null or undefined check instead (bueno has a utility function for this)